### PR TITLE
Send usage message when a user starts the bot

### PR DESCRIPTION
**Pull request by marcofranzoni**
Hi @rob93c what do you think of think of this simple solution?

### CODE REVIEW

**Comment by rob93c** [comment]
Hi @marcofranzoni and thanks for your work!
I'll look into your changes right away

File: src/main/java/com/github/stickerifier/stickerify/telegram/model/TelegramRequest.java [hunk 1]
```diff
@@ -74,7 +74,7 @@ public Integer getMessageId() {
 	public String getDescription() {
 		var description = "request from " + getUsername();
 
-		if (START_COMMAND.equals(message.text())) {
+		if (START_COMMAND.equalsIgnoreCase(message.text())) {
```
Comment (rob93c) [inline src/main/java/com/github/stickerifier/stickerify/telegram/model/TelegramRequest.java:77]: Why did you change this check? The start command is triggered by Telegram when a user activates the bot and it will always be `/start`

File: src/main/java/com/github/stickerifier/stickerify/telegram/model/TelegramRequest.java [hunk 2]
```diff
@@ -88,11 +88,12 @@ private String getUsername() {
...
 
 	public Answer getAnswerMessage() {
-		return isHelpCommand() ? HELP : ABOUT;
+		return isHelpOrStartCommand() ? HELP : ABOUT;
 	}
 
-	private boolean isHelpCommand() {
-		return HELP_COMMAND.equalsIgnoreCase(message.text());
+	private boolean isHelpOrStartCommand() {
+		return HELP_COMMAND.equalsIgnoreCase(message.text()) ||
```
Comment (rob93c) [inline src/main/java/com/github/stickerifier/stickerify/telegram/model/TelegramRequest.java:95]: To be consistent with the rest of the project, you should put the binary operator at the beginning of the next line

**Review by rob93c — CHANGES_REQUESTED** [review]

File: src/main/java/com/github/stickerifier/stickerify/telegram/model/TelegramRequest.java (hunk 1 above)
Comment (marcofranzoni) [reply src/main/java/com/github/stickerifier/stickerify/telegram/model/TelegramRequest.java:77]: Just to be consistent with the equalsIgnoreCase already present in the code, but your point of view makes sense

**Review by rob93c — APPROVED** [review]
